paper/paper.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "SWORDS: A framework for the Scan and revieW of Open Research Data and Software"
+tags:
+  - Python
+  - data analysis
+  - FAIR
+  - open science
+authors:
+  - name: Keven Quach
+    orcid: 0009-0002-7017-0331
+    equal-contrib: true
+    corresponding: true 
+    affiliation: 1
+  - name: Jonathan de Bruin
+    orcid: 0000-0002-4297-0502
+    equal-contrib: true
+    affiliation: 2
+  - name: Anna-Lena Lamprecht
+    orcid: 0000-0003-1953-5606
+    equal-contrib: true
+    affiliation: 3
+affiliations:
+ - name: Independent Researcher, Germany
+   index: 1
+ - name: Utrecht University, Netherlands
+   index: 2
+ - name: University of Potsdam, Germany
+   index: 3
+date: 18 June 2023
+bibliography: paper.bib
+
+---
+
+# Summary
+
+SWORDS (Scan and revieW of Open Research Data and Software) is a framework designed to provide insights into an organization's open-source activities through a structured approach. The framework is divided into three core stages that can be executed independently:
```

Comment:
It might be good to explain why the stages can be executed independently. This might confuse the reader. For example, stage 1 is relevant to organisations not having an overview of the GitHub profiles of their staff. If so, they can execute stage 1.

And variations